MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -6118,7 +6121,9 @@ public string GetVideotoolboxVidDecoder(EncodingJobInfo state, EncodingOptions o
                                     && state.VideoStream.VideoRange == VideoRange.HDR
                                     && (state.VideoStream.VideoRangeType == VideoRangeType.HDR10
                                         || state.VideoStream.VideoRangeType == VideoRangeType.HLG
-                                        || (state.VideoStream.VideoRangeType == VideoRangeType.DOVI
+                                        || ((state.VideoStream.VideoRangeType == VideoRangeType.DOVI
+                                            || state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHDR10
+                                            || state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHLG)
```

Comment:
```suggestion
                                        || (state.VideoStream.VideoRangeType == VideoRangeType.DOVI
```

In VideoToolbox the `useOclToneMapping` is only enabled for DOVI Profile 5. No need to change here.

Reply:
I *think* it could be needed, as the user might have explicitly disabled VideoToolbox's native tone mapping. If `VideoRangeType.DOVI` does not imply `VideoRangeType.DOVIWithHDR10` or `VideoRangeType.DOVIWithHLG`, then users with such configurations might not have hwsurface initialized, which could cause hwmap to fail.

Reply:
> If so there is no reason to exclude VideoRangeType.{HDR10,HLG}.
They are already included, the code segment shown here might be incomplete, but all kind of HDR is already included
https://github.com/jellyfin/jellyfin/blob/6b4951fcf5c0fe2a473b2c13367fdf367a4a4c78/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L6121-L6128

Reply:
Then this is necessary. I read the wrong line.